src/css/base/lists.scss

```diff
@@ -0,0 +1,6 @@
+
+ul,
+ol {
+  margin: 1rem 0 1rem 0.9em;
```

Comment:
Should we change this value to `@include margin(1em null);` to match the upcoming fix?